### PR TITLE
Fixing issue #44

### PR DESCRIPTION
**Pull request by Badoche1337**
Adding problem and change handling in actors button removal feature. Removing limitation on "group_ticket" check in conditional inclusion of JS file.

### CODE REVIEW

File: setup.php [hunk 1]
```diff
@@ -62,6 +62,14 @@ function plugin_init_escalade() {
                }
             }
 
+            // on ticket page (in edition)
+            if ((strpos($_SERVER['REQUEST_URI'], "ticket.form.php") !== false
+               || strpos($_SERVER['REQUEST_URI'], "problem.form.php") !== false
+               || strpos($_SERVER['REQUEST_URI'], "change.form.php") !== false) && isset($_GET['id'])) {
```
Comment (orthagh) [inline setup.php]: globally ok with the pr.
But here, il woul be better to check if we really need to load the js file.
If none of hide buttons checkboxes enabled, don't load it

**Comment by Badoche1337** [comment]
Done

**Comment by orthagh** [comment]
So last thing, fix CS issue and we're done:
https://travis-ci.org/pluginsGLPI/escalade/jobs/346816228#L560-L561

**Comment by Badoche1337** [comment]
My bad forget that ! that's ok now

**Pull request merged**